keps/sig-network/2433-topology-aware-hints/kep.yaml

```diff
@@ -34,7 +34,7 @@ latest-milestone: "v1.26"
 milestone:
   alpha: "v1.21"
   beta: "v1.23"
-  stable: "v1.26"
+  stable: "v1.27"
```

Comment:
If you're planning to graduate to stable in 1.27, please change the line 31

The current changes looks fine from PRR perspective, but if something will visibly change, please ping me.

Reply:
Thanks! We ended up punting to 1.28.